test/performance/odp_l2fwd.c

```diff
@@ -2261,7 +2245,7 @@ int main(int argc, char *argv[])
 				     thr_param, num_workers);
 
 	if (num_thr != num_workers) {
-		ODPH_ERR("Error: worker create failed %i\n", num_thr);
+		ODPH_ERR("Worker create failed %i\n", num_thr);
```

Comment:
":" after "failed" to match the new style.

Reply:
Fixed in v2